src/components/Disclosure/__tests__/Disclosure.test.tsx

```diff
@@ -18,7 +16,8 @@ describe.only('Disclosure', () => {
     });
 
     test.each(new Array<DisclosureSize>('m', 'l', 'xl'))('render with given "%s" size', (size) => {
-        render(<Disclosure qa={qaId} size={size} />);
+        const qaId = 'disclosure';
```

Comment:
Why we need any changes in tests?

Reply:
without this qa props, there will be no data-qa attribute on the div
![image](https://github.com/user-attachments/assets/5272b356-c17e-4766-98e0-0a420737ac0a)

Reply:
My point is that your changes should work without changing the tests, otherwise it's still a breaking change